src/components/pacbio/PacbioRunWellDefaultEdit.vue

```diff
@@ -205,12 +167,12 @@ export default {
         '24.0',
         '30.0',
       ],
-      generateHifiOptions: {
```

Comment:
If we are only supporting Sequel IIe now we also need to remove the ability to select it [from here](https://github.com/sanger/traction-ui/blob/develop/src/components/pacbio/PacbioRunInfoEdit.vue#L112)